modules/main.rb

```diff
@@ -10,7 +10,7 @@ def choose_option
     puts '4 - Create a book'
     puts '5 - Create a rental'
     puts '6 - List all rentals for a given person id'
-    puts '7 - Exit'
+    puts '7 - Exit!'
   end
 
```

Comment:
- [ ] Kindly ensure that we move all request for input and input validations from the `app.rb` file into the `main.rb` file, to follow the Single Responsiblity Rule, YAGNI, KISS and SOLID [lessions](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), all request for input and input validations should be placed in the `main.rb` file. For example, lets us take one of the methods `create_teacher` in line `89-94` all these are input and input validation that should be moved into the `main.rb` file. **NB**: I only used this method as an example, kindly use this same principle in all
- [ ] Let us kindly move `teacher = Teacher.new(age, name, specialization)` and `student = Student.new(age, name, parent_permission)` into the the `create_person` method